task_5/Toma.Lobanova/src/main/java/ua/od/hillel/BinaryNumber/BinaryNumber.java

```diff
@@ -0,0 +1,21 @@
+package ua.od.hillel.BinaryNumber;
+
+
+public class BinaryNumber {
+    public static void main(String...args){
+        int number=3456;
+        StringBuilder builder=new StringBuilder();
+        int pow = 0;
+        for (int i = 2; i <= number; i *= 2, pow ++) {
```

Comment:
That is just to count the maximum pow of 2. I can add pow++ to the body but as an artist i should say that is more beautiful ;) I can fix it if you want but i like it more